src/providers/twitch/TwitchChannel.cpp

```diff
@@ -797,6 +806,11 @@ bool TwitchChannel::isLive() const
     return this->streamStatus_.accessConst()->live;
 }
 
+bool TwitchChannel::isRerun() const
+{
+    return this->streamStatus_.accessConst()->live;
```

Comment:
This feels wrong

Reply:
Oh lmao, that's the worst copypaste-o I've done in a while